docs/development-guide.md

```diff
@@ -44,6 +44,26 @@ To build ceph-csi in a container:
 
 The built binary will be present under `_output/` directory.
 
+### Running Ceph-CSI tests
+
+Once the changes to the sources compile, it is good practise to run the tests
+that validate the style and other basics of the source code. Execute the unit
+tests (in the `*_test.go` files) and check the formatting of YAML files and
+shell scripts:
+
+`$ make containerized-test`
```

Comment:
this is more specific containerized tests. need to update the title or put a subtitle as `running containerized test`

Reply:
Yeah, that makes sense. I was not going to explain how to run the tests outside of a container as it requires installing quite some dependencies. Title has been updated.